packages/react-dom-bindings/src/client/ReactDOMFloatClient.js

```diff
@@ -102,6 +103,10 @@ export function cleanupAfterRenderResources() {
   previousDispatcher = null;
 }
 
+export function prepareToCommitHoistables() {
+  refreshedCaches.clear();
```

Comment:
Setting it to null should be faster. Especially if it's already empty and it clears out over allocated space that we'll never use again.

Reply:
It's slightly different now but the tagCaches get reset each render and they are nulled out instead of cleared